src/Analyzers/Core/Analyzers/Helpers/AnalyzerHelper.cs

```diff
@@ -4,18 +4,19 @@
 
 #nullable enable
 
+using Microsoft.CodeAnalysis.Options;
+
 #if CODE_STYLE
-using Microsoft.CodeAnalysis.Internal.Options;
+using TOption = Microsoft.CodeAnalysis.Options.IOption2;
```

Comment:
A rare case where `#if CODE_STYLE` in a helper methods file is helping avoid duplicate code, which needs to operate on core fields of `IOption` and `IOptions`.